nix/tests/sql/enable_tracking.sql

```diff
@@ -0,0 +1,2 @@
+alter role postgres set track_functions = 'pl';
+alter role postgres set track_io_timing = on;
```

Comment:
Let's reset it in the same test so it doesn't affect other tests
```suggestion
alter role postgres set track_functions = 'pl';
alter role postgres set track_io_timing = on;

alter role postgres reset track_functions;
alter role postgres reset track_io_timing;
```